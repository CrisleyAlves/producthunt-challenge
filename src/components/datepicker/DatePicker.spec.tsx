import React from "react";
import { render } from "@testing-library/react";
import { describe, it, expect, vi } from "vitest";

import InputDate from "./DatePIcker";

describe("InputDate", () => {
  it("renders the DatePicker component", () => {
    const { container } = render(
      <InputDate onChangeDate={vi.fn()} currentDate={new Date("2023-01-01")} />
    );

    expect(container).toMatchSnapshot();
  });
});
