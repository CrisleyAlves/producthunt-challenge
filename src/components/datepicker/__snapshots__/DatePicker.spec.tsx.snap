// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`InputDate > renders the DatePicker component 1`] = `
<div>
  <div
    class="react-datepicker-wrapper"
  >
    <div
      class="react-datepicker__input-container"
    >
      <input
        class="datepicker"
        type="text"
        value="12/31/2022"
      />
    </div>
  </div>
</div>
`;
